bin/graph_altimeter_batch.py

```diff
@@ -49,26 +49,12 @@ def run_scan():
         accounts = accounts_to_scan.split(",")
     else:
         accounts = get_aws_accounts(asset_inventory_api_url)
-    accounts_per_batch = os.getenv("ACCOUNTS_BATCH", None)
```

Comment:
We should also remove `ACCOUNTS_BATCH` from [/env/local.env](https://github.com/adevinta/graph-altimeter/blob/7cf9e567efc12856a958b92cd554d240a5c5a679/env/local.env#L11-L12)

Reply:
good catch thank you.